### PR TITLE
[openui5] README: clarification about referenced types

### PR DESCRIPTION
**Pull request by akudev**
This is a README-only change.

### CODE REVIEW

File: types/openui5/README.md [hunk 1]
```diff
@@ -42,10 +42,34 @@ The reasoning behind this is that with semantic versioning, the API will remain
...
 * For perfectly matching definitions, use the version of [`@openui5/ts-types-esm`](https://www.npmjs.com/package/@openui5/ts-types-esm) which is identical to the used OpenUI5 version.
 
 
+## jQuery and QUnit References and their Versions
+
+OpenUI5 includes copies of jQuery and QUnit and occasionally exposes types from them in its API.
+
+In the DefinitelyTyped infrastructure, we can only specify *that* the OpenUI5 types/APIs depend on the types of jQuery and QUnit, but we cannot specify the respective *version*. So the `package.json` file generated by DefinitelyTyped for the npm release references the `"*"` version (any/latest version), not the version of the jQuery/QUnit copy which is actually included with the OpenUI5 framework.
+
+Hence, as of writing (OpenUI5 version 1.100, March 2022), the versions of the types being installed together with `@types/openui5` are:
```
Comment (codeworrior) [inline types/openui5/README.md:55]: today is the last chance to submit this wording...

**Review by codeworrior — APPROVED** [review]
LGTM

File: types/openui5/README.md (hunk 1 above)
Comment (petermuessig) [reply types/openui5/README.md:55]: Tomorrow still works, isn't it?

**Review by petermuessig — APPROVED** [review]
Approve

File: types/openui5/README.md (hunk 1 above)
Comment (codeworrior) [reply types/openui5/README.md:55]: Fixed.

**Pull request merged**